BlockchainSdkTests/Common/AddressTests.swift

```diff
@@ -1217,5 +1217,52 @@ class AddressesTests: XCTestCase {
     
     // MARK: - Radiant
     
-    // TODO: - Insert Radiant basic address tests
+    // Validate by https://github.com/RadiantBlockchain/radiantjs
+    func testRadiantAddressGeneration() throws {
+        let addressServiceFactory = AddressServiceFactory(blockchain: .radiant(testnet: false))
+        let addressService = addressServiceFactory.makeAddressService()
+        
+        let addr1 = try addressService.makeAddress(from: secpCompressedKey)
+        XCTAssertEqual(addr1.value, "1JjXGY5KEcbT35uAo6P9A7DebBn4DXnjdQ")
+        
+        let addr2 = try addressService.makeAddress(from: secpDecompressedKey)
+        XCTAssertEqual(addr2.value, "1JjXGY5KEcbT35uAo6P9A7DebBn4DXnjdQ")
+        
+        let anyOnePublicKey = Data(hexString: "039d645d2ce630c2a9a6dbe0cbd0a8fcb7b70241cb8a48424f25593290af2494b9")
+        let addr3 = try addressService.makeAddress(from: anyOnePublicKey)
+        
+        XCTAssertEqual(addr3.value, "12dNaXQtN5Asn2YFwT1cvciCrJa525fAe4")
+        
+        let anyTwoPublicKey = Data(hexString: "03d6fde463a4d0f4decc6ab11be24e83c55a15f68fd5db561eebca021976215ff5")
+        let addr4 = try addressService.makeAddress(from: anyTwoPublicKey)
+        
+        XCTAssertEqual(addr4.value, "166w5AGDyvMkJqfDAtLbTJeoQh6FqYCfLQ")
+        
+        // For ed25519 wrong make address from public key
+        let edPublicKey = Data(hex: "e7287a82bdcd3a5c2d0ee2150ccbc80d6a00991411fb44cd4d13cef46618aadb")
+        let addrEdv = try? addressService.makeAddress(from: edPublicKey)
+        XCTAssertNil(addrEdv)
```

Comment:
Может лучше `XCTAssertThrowsError`

```suggestion
       XCTAssertThrowsError(try addressService.makeAddress(from: edPublicKey)
```